### PR TITLE
[13.0][MIG] fieldservice_purchase

### PR DESCRIPTION
**Pull request by baimont**
Migration of fieldservice_purchase to 13.0

https://github.com/OCA/field-service/issues/354

### CODE REVIEW

**Comment by baimont** [comment]
I understand that this branch is red because of code coverage. And the branch needs tests.
But I'm wondering, why weren't there any test? Was the branch green for v12.0 without test?

**Comment by brian10048** [comment]
@baimont We do not have tests for all modules, which is why many are still in Beta status in the manifest.  If you are able to add tests to this module, it would be greatly appreciated!

File: fieldservice_purchase/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+To use this module, you need to:
+
+* Go to Purchases > Purchase > Vendor Pricelist
```
Comment (brian10048) [inline fieldservice_purchase/readme/USAGE.rst]:
```suggestion
* Go to Purchases > Configuration > Vendor Pricelist
```

**Review by brian10048 — CHANGES_REQUESTED** [review]

**Review by brian10048 — APPROVED** [review]

**Review by marcelsavegnago — APPROVED** [review]
LGTM

**Review by hparfr — APPROVED** [review]

**Pull request merged**